### ISSUE TITLE
[A11y]: Contrast and other issues.

### ISSUE THREAD

**Issue by codewordcreative** [passkeydeveloper/whatarepasskeys.info#47]
### Page URL / Component

https://whatarepasskeys.info

### Barrier Category

Color Contrast / Visual Legibility

### Assistive Technology Used

N/A

### Detailed Description

I noticed the contrast issues instantly because I am contrast-sensitive myself. This persisted in dark and light mode, especially with the use of small grey text for critical details. 

The WAVE report reveals further issues I cannot comment on: https://wave.webaim.org/report#/https://whatarepasskeys.info
